packages/graphql/src/schemaTypes/objectTypes/gql-Query.ts

```diff
@@ -107,6 +107,11 @@ export const Query = objectType({
       resolve: (_, args, ctx) => ctx.coreData.scaffoldedFiles,
     })
 
+    t.nonNull.boolean('invokedFromCli', {
+      description: 'Whether the app was invoked from the CLI',
```

Comment:
```suggestion
      description: 'Whether the app was invoked from the CLI', False if user is using the binary without npm.
```

Just imagining coming in on this later and not being sure how else the app can be invoked.